lib/improver/tests/lapse_rate/Test_LapseRate.py

```diff
@@ -0,0 +1,367 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2018 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Unit tests for the LapseRate plugin."""
+
+import numpy as np
+import unittest
+
+import cf_units
+from iris.cube import Cube
+from iris.tests import IrisTest
+from iris.coords import (DimCoord,
+                         AuxCoord)
+
+from improver.grids import STANDARD_GRID_CCRS
+
+from improver.lapse_rate import LapseRate
+
+
+def reset_cube_data(temperature_cube, orography_cube, land_sea_mask_cube):
+    """ Resets the cube data to its defaults. """
+    temperature_cube.data[:, :, :] = 0
+    orography_cube.data[:] = 0
+    land_sea_mask_cube.data[:] = 1
+
+
+class Test__repr__(IrisTest):
+    """Test the repr method."""
+
+    def test_basic(self):
+        """Test that the __repr__ returns the expected string."""
+        result = str(LapseRate())
+        msg = ('<LapseRate>')
+        self.assertEqual(result, msg)
+
+
+class Test_calc_lapse_rate(IrisTest):
```

Comment:
This should have a double underscore to denote a private class function: `Test__calc_lapse_rate`.